src/www/index.html

```diff
@@ -349,6 +349,12 @@
             <input class="form-control" style="width: 49.2%; display: inline" id="ftp-host" placeholder="host" type="text"> <input class="form-control" style="width: 49.2%; display: inline" id="ftp-port" placeholder="port" type="text"> <input class="form-control" id="ftp-user" placeholder="username" style="width: 49.2%; display: inline" type="text"> <input class="form-control" id="ftp-pass" placeholder="password" style="width: 49.2%; display: inline" type="password"> <input class="form-control" id="ftp-path" placeholder="/path/to/deploy/folder" type="text">
           </div>
         </section>
+        <section>
+          <input id="enable-google-analytics" title="Enable Google Analytics" type="checkbox"> <label for="enable-google-analytics">Enable Google Analytics</label>
+          <div id="ganalytics-id-container" style="display: none">
+            <input class="form-control" style="width: 49.2%; display: inline" id="ganalytics-id" placeholder="google analytics id" type="text">
```

Comment:
Add validations. User is able to generate the app even when the field is empty. Change placeholder to `Your google analytics ID`.

Reply:
@geekyd user should be able to generate app even without google analytics, this is because it depends on his choice if he wants it or not.

Reply:
If the user selects the option to add google analytics, in this case the input must be a valid ID.

Reply:
@geekyd that validation can't be made, since the `google analytics`code analyzes only when the site is active. When a user opens that page in his/her browser then that code makes a  request to the google giving information about sessions, page that is being viewed etc. And also capturing those request is not possible so there is no way to verify `id`. And if a person is using google analytics functionality then he must be using entering the correct one.

Reply:
@geekyd